Source/WebCore/css/CSSSelector.h

```diff
@@ -157,6 +157,7 @@ class CSSSelector {
     bool matchesPseudoElement() const;
     bool isSiblingSelector() const;
     bool isAttributeSelector() const;
+    bool isHostPseudoClass() const;
```

Comment:
We should add helpers like `isPseudoClass(PseudoClass)`,` isPseudoElement(PseudoElement)` etc. at some point